core/src/model/monsters/Monster.java

```diff
@@ -0,0 +1,37 @@
+package model.monsters;
+
+import model.LivingEntity;
+import model.Entity;
+import com.dongbat.jbump.World;
+
+public abstract class Monster extends LivingEntity {
+    //Bad practice? only works if static
+    public static final String DEFAULTFACTION = "monsters";
```

Comment:
maybe better to use another way to determine the defualt faction?

Reply:
Have you considered creating an enum for the different factions?